src/Tides2.cpp

```diff
@@ -103,8 +103,8 @@ struct Tides2 : Module {
 		config(NUM_PARAMS, NUM_INPUTS, NUM_OUTPUTS, NUM_LIGHTS);
 		configParam(RANGE_PARAM, 0.0, 1.0, 0.0, "Frequency range");
 		configParam(MODE_PARAM, 0.0, 1.0, 0.0, "Output mode");
-		configParam(FREQUENCY_PARAM, -48, 48, 0.0, "Ramp mode");
-		configParam(SHAPE_PARAM, 0.0, 1.0, 0.5, "Frequency");
+		configParam(FREQUENCY_PARAM, -48, 48, 0.0, "Frequency");
+		configParam(SHAPE_PARAM, 0.0, 1.0, 0.5, "Ramp mode");
 		configParam(RAMP_PARAM, 0.0, 1.0, 0.0, "Shape");
```

Comment:
Doesn't ramp and shape need to be changed too?

Reply:
Oh, yes that's right.

Reply:
I added another commit to swap ramp mode and shape.